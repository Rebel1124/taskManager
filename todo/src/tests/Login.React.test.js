//Import the required Libraries
import React from 'react';
import Login from './Log.js';
import renderer from 'react-test-renderer';

//We then setup the test function, which is a snapshot test.
test('renders correctly', () => {
    const tree = renderer
    .create(<Login />)
    .toJSON();
    expect(tree).toMatchSnapshot();
    });