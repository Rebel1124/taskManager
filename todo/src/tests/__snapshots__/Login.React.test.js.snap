// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="container"
>
  <br />
  <h1
    className="text-info"
  >
    Login Details
  </h1>
  <br />
  <form>
    <div
      className="form-group"
    >
      <input
        autoComplete="off"
        className="form-control"
        name="username"
        onChange={[Function]}
        placeholder="username"
        value=""
      />
    </div>
    <br />
    <div
      className="form-group"
    >
      <input
        autoComplete="off"
        className="form-control"
        name="password"
        onChange={[Function]}
        placeholder="password"
        type="password"
        value=""
      />
    </div>
    <br />
    <div
      className="form-inline"
    >
      <div
        style={
          Object {
            "padding": "10px",
          }
        }
      >
        <button
          className="btn btn-lg btn-info"
          onClick={[Function]}
        >
          Submit
        </button>
      </div>
      <div
        style={
          Object {
            "padding": "10px",
          }
        }
      >
        <button
          className="btn btn-lg btn-secondary"
          onClick={[Function]}
        >
          Authenticate
        </button>
      </div>
    </div>
  </form>
</div>
`;
